frontend/src/components/custom-tools/document-parser/DocumentParser.jsx

```diff
@@ -242,9 +249,14 @@ function DocumentParser({
       prompt_id: details.prompts[startIndex]?.prompt_id,
     };
 
+    let url = promptUrl("reorder");
```

Comment:
@tahierhussain trailing "/" is not required for this? i believe we are following like every url should have trailing "/".

Reply:
@vishnuszipstack  This API does not require a trailing slash, similar to all the SPS APIs. I recall some of the team members were planning to remove the trailing slash from all APIs as it is unnecessary.

@chandrasekharan-zipstack @Deepak-Kesavan Please let us on the status of that, and I will make the changes accordingly.

Reply:
I suggest we remain consistent and add it for all - we can take a call to remove it altogether in all APIs if necessary. Doing this for a few APIs alone can be confusing

Reply:
Done, @chandrasekharan-zipstack @vishnuszipstack. The trailing slash has been added to all the SPS related APIs.